index.html

```diff
@@ -29,31 +29,185 @@
     </header>
 
     <!-- main section -->
-    <main class="main-section">
-      <div class="content">
-        <p class="author">
-          I’m Ismail <br />
-          Glad to see <br />
-          you!
-        </p>
-        <p class="article">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
+    <main>
+      <!-- navbar headline features -->
+      <div class="headline-section">
+        <div class="content">
+          <p class="author">
+            I’m Ismail <br />
+            Glad to see <br />
+            you!
+          </p>
+          <p class="article">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+
+        <div class="connect">
+          <p>LET’S CONNECT</p>
+        </div>
+
+        <div class="social-icons">
+          <span class="iconify" data-icon="iconoir:github"></span>
+          <span class="iconify" data-icon="eva:linkedin-fill"></span>
+          <span class="iconify" data-icon="fontisto:angelist"></span>
+          <span class="iconify" data-icon="ei:sc-twitter"></span>
+          <span class="iconify" data-icon="bx:bxl-medium"></span>
+        </div>
+      </div>
+
+      <!-------------------------------------- middle-section ---------------------------------------->
+      <!-- tonic section 1 -->
+      <div class="tonic-section1">
```

Comment:
- [ ] The container of your projects section should be a `section` tag, you are currently using divs for each project, kindly wrap your projects in a `section` tag.

- [ ] The main container of your projects section should be aligned and positioned using `display: grid`, after wrapping your projects in a `section` tag, add the grid styling to it.

- [ ] Please make sure to add `background-color` for each project and the project wrapper that matches the Figma design.

- [ ] The content of each project should be aligned using `flexbox`, currently, it's just a div element that has block display by default.